lexibank_chaconcolumbian.py

```diff
@@ -19,43 +19,53 @@ class Dataset(BaseDataset):
     id = "chaconcolumbian"
```

Comment:
We agreed yesterday on the name "CustomConcept" instead of custom names for custom concepts. Could you change this please?